nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -420,18 +424,24 @@ def _impl(inputs, attr, params):
             ignores=['index_type', 'T'])(new_inputs, attr)
     return _impl
 
+def _split():
+    def _impl(inputs, attr, params):
+        pop_node = inputs.pop(0)
```

Comment:
Ref. [Split](https://www.tensorflow.org/api_docs/python/tf/split)
tensorflow has two variations for split as ```num_split``` and ```size_split```.
Please handle if possible or raise exception accordingly.

If handling size_split please handle ```num``` attribute if possible or raise exception accordingly.

Reply:
[Split](https://www.tensorflow.org/versions/r1.0/api_docs/cc/class/tensorflow/ops/split) had num_split attribute and [SplitV](https://www.tensorflow.org/versions/r1.0/api_docs/cc/class/tensorflow/ops/split-v) had size_splits attribute. As of this PR supported [Split](https://www.tensorflow.org/versions/r1.0/api_docs/cc/class/tensorflow/ops/split-v) operator